docs/design/index.md

```diff
@@ -40,8 +40,16 @@ The downside of the microautonomy architecture is that the computational perform
 
 ### Autoware concepts
 
+The [Autoware concepts page](autoware-concepts/index.md) describes the "design philosophy" of Autoware. Readers (service providers and all Autoware users) will know the basic concepts underlying Autoware development, such as microautonomy and core/universe architecture.
```

Comment:
I don't think there is any need for the double quotes around `design philosophy`, `overview` nor `detailed`.

```suggestion
The [Autoware concepts page](autoware-concepts/index.md) describes the design philosophy of Autoware. Readers (service providers and all Autoware users) will learn the basic concepts underlying Autoware development, such as microautonomy and the Core/Universe architecture.
```

Reply:
Thank you. Addressed.